Dockerfile

```
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.21.1 AS builder
+FROM golang:1.22.0 AS builder
```

```suggestion
FROM golang:1.22.5 AS builder
```
go v1.22.0 has critical CVEs